ethereum/contracts/ETHApp.sol

```diff
@@ -36,6 +49,16 @@ contract ETHApp is Application {
         totalETH = totalETH.add(msg.value);
 
         emit AppTransfer(msg.sender, _recipient, msg.value);
```

Comment:
Do we still want to emit that?

Reply:
could be useful for things like block explorers and whatever else, though can remove later on if we need to hyper optimize for gas

Reply:
yeah is useful for informational purposes. The substrate side also emits an informational `Minted` event. 

I would rename it to `Locked` though.